packages/frontend2/src/app/(new)/(other)/bridges/risk/page.tsx

```diff
@@ -0,0 +1,40 @@
+import { BridgesMvpWarning } from '~/app/_components/bridges-mvp-warning'
+import { SimplePageHeader } from '~/app/_components/simple-page-header'
+import { getBridgeRiskEntries } from '~/server/features/bridges/get-bridge-risk-entries'
+import { getImplementationChangeReport } from '~/server/features/implementation-change-report/get-implementation-change-report'
+import { getLatestTvlUsd } from '~/server/features/tvl/get-latest-tvl-usd'
+import { getProjectsVerificationStatuses } from '~/server/features/verification-status/get-projects-verification-statuses'
+import { getDefaultMetadata } from '~/utils/get-default-metadata'
+import { BridgesFilterContextProvider } from '../_components/bridges-filter-context'
+import { BridgesRiskTables } from './_components/table/bridges-risks-tables'
+
+export const metadata = getDefaultMetadata({
+  openGraph: {
+    url: '/bridges/summary',
+  },
+})
+
+export default async function Page() {
+  const [tvl, projectsVerificationStatuses, implementationChangeReport] =
+    await Promise.all([
+      getLatestTvlUsd({ type: 'bridge' }),
+      getProjectsVerificationStatuses(),
+      getImplementationChangeReport(),
+    ])
```

Comment:
We need to decide which pattern to use:
* fetch everything from db in page and pass to get entries fn
* fetch everything from db in get entries and on page just call get entries fn

Personally I would opt for 2nd approach